app/screens/account_notifications/account_notifications.js

```diff
@@ -138,7 +138,7 @@ class AccountNotifications extends PureComponent {
         if (this.state) {
             this.setState(newState);
         } else {
-            this.state = {...newState};
+            this.state = {...newState}; // eslint-disable-line
```

Comment:
Shouldn't we just set this.state in the constructor so that we don't need to set it directly here?

Reply:
This whole file gets removed in this PR https://github.com/mattermost/mattermost-mobile/pull/879 so I just disabled it to pass tests.